lang/java/reef-runtime-local/src/main/java/org/apache/reef/runtime/local/client/DriverConfigurationProvider.java

```diff
@@ -35,12 +36,15 @@
 
   private final int maxEvaluators;
   private final double jvmHeapSlack;
+  private final Set<ConfigurationProvider> configurationProviders;
 
   @Inject
   DriverConfigurationProvider(final @Parameter(MaxNumberOfEvaluators.class) int maxEvaluators,
-                              final @Parameter(JVMHeapSlack.class) double jvmHeapSlack) {
+                              final @Parameter(JVMHeapSlack.class) double jvmHeapSlack,
+                              final @Parameter(DriverConfigurationProviders.class) Set<ConfigurationProvider> configurationProviders)  {
```

Comment:
Why do we need this in addition to the merge that happens in `REEFImplementation`?